benchmarks/kubectl-mtb/internal/kubectl-mtb/get.go

```diff
@@ -27,9 +27,22 @@ import (
 )
 
 var getCmd = &cobra.Command{
-	Use:   "get",
+	Use:   "get <resource>",
 	Short: "Display one or many benchmarks.",
 
+	PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) == 0 {
+			return fmt.Errorf("Please specify any resource")
+		}
+		if !supportedResourceNames.Has(args[0]) {
+			return fmt.Errorf("Please specify any valid resource")
```

Comment:
Change to `Please specify a valid resource`.

Assuming we are showing the user the list of valid resources, right?